### ISSUE TITLE
Dataset list under Los Datos tab are not translated to Spanish 

### ISSUE THREAD

**Issue by gdaldegan** [ConservationInternational/trends.earth#675]
![image](https://user-images.githubusercontent.com/22821787/178811201-ca3eb757-2128-4dca-8c4c-fabb6822ba86.png)

**Comment by azvoleff** [comment]
@gdaldegan is this still an issue? I believe this is now fixed

**Comment by gdaldegan** [comment]
@azvoleff yes, it is still an issue! Installed version 2.1.7 (fe2e0aa4 2022/12/12) and datasets are not translated to Spanish nor to Portuguese. Actually, there other parts that still need to be addressed, such as the explanatory notes for functions, the Reset Land Cover Legend warning - see below:

![Screenshot_20221213_114737](https://user-images.githubusercontent.com/22821787/207469755-519bce93-403e-4de9-a23d-c0d1cf5f571e.png)

**Comment by gdaldegan** [comment]
Consistently getting a **TypeError 'int' object is not iterable** Py Error when changing language of QGIS' User interface translation, even though QGIS interface does change the language after restarting it.
